api/src/request.rs

```diff
@@ -93,7 +94,13 @@ where
         return Err(JsonRpcError::missing_method(request));
     }
 
-    match method {
+    let method_str = format!("{method:?}");
```

Comment:
Instead of using the debug format of the `MethodName` enum, maybe it would be better to use the string value from the JSON directly. The JSON method name is prescribed by the Ethereum RPC spec, while our internal representation of it may change in the future. So think the metric label will be more stable if the JSON value is used.

Something like

```rust
let method_str = json_utils::get_field(&request, "method")
    .as_str()
    .ok_or(JsonRpcError::missing_method(request.clone()))?;

let method: MethodName = method_str.parse()?;
```